### PR TITLE
RSA Netwitness Bug Fixes

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
Ready

## Related Issues
fixes: demisto/etc#19041
fixes: demisto/etc#18744
fixes: demisto/etc#19337

## Description
Fixing some various issues. See the related issues for full scope.

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/558bf024a763895d4ba670d0ba1dc90b2acef5ca/Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/558bf024a763895d4ba670d0ba1dc90b2acef5ca/Integrations/RSANetWitness_v11_1/CHANGELOG.md)

### CODE REVIEW

File: Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def get_token_request(username, password):
         'NetWitness-Version': VERSION
     }
 
-    response = requests.post(url, headers=get_token_headers, data=username_password, verify=USE_SSL)
+    response = requests.post(url, headers=get_token_headers, data=username_password, verify=USE_SSL, proxies=PROXIES)
```
Comment (yuvalbenshalom) [inline Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py]: please move the proxy handling to be standard(handle_proxy, without passing argument) if there is no proof that it helps. as this is why it looks like from resent issue updates

**Comment by ShahafBenYakir** [comment]
@amshamah419 I added the fix that seemed to solve US bank issue. Can you handle the PR from here?

**Comment by amshamah419** [comment]
Yeah I can

**Comment by amshamah419** [comment]
@ShahafBenYakir

**Comment by ronykoz** [comment]
what's the eta for this?
@yuvalbenshalom @amshamah419

**Comment by amshamah419** [comment]
@ronykoz Needs a final review and then for docs approval, otherwise it's ready to be merged.

File: Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py [hunk 2]
```diff
@@ -611,7 +617,7 @@ def fetch_incidents():
     LOG('Fetching incidents since {}'.format(timestamp))
     netwitness_incidents = get_all_incidents(
         since=timestamp,
-        limit=100
+        limit=FETCH_LIMIT
```
Comment (ronykoz) [inline Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py]: Not sure that the limit should be here, it should be in the incident creation side, we can miss incidents this way

Comment (amshamah419) [reply Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.py]: I'm not sure I follow. We have a limit on how many incidents can be pulled in at a time when `fetch_incidents` runs. It's user configurable in the integration settings.

**Comment by amshamah419** [comment]
https://github.com/demisto/content/blob/rsa-netwitness-bugfix/Integrations/RSANetWitness_v11_1/CHANGELOG.md - Can you take a look at the RNs and [this line here?](https://github.com/demisto/content/blob/rsa-netwitness-bugfix/Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml#L48) @kirbles19

File: Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml [hunk 3]
```diff
@@ -44,6 +44,11 @@ configuration:
   name: fetch_time
   required: false
   type: 0
+- defaultvalue: 100
+  display: Maxmimum amount of incidents to ingest in one fetch cycle.
```
Comment (yaakovi) [inline Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1.yml]:
```suggestion
  display: 'Fetch limit: the maximum number of incidents to fetch'
```

**Comment by kirbles19** [comment]
@amshamah419 Updated the changelog, the line for fetch limit display name looks fine.

**Pull request merged**